lib/ingram_micro/elements/return_authorization_shipment_information.rb

```diff
@@ -0,0 +1,35 @@
+module IngramMicro
+  class ReturnAuthorizationShipmentInformation < BaseElement
+    DEFAULTS = IngramMicro::OutboundShipmentInformation::DEFAULTS
+
+    def defaults
+      DEFAULTS
+    end
+
+    def valid_shipping_methods?
```

Comment:
same as the comment in `outbound_shipment_information.rb`. in fact, these two classes share quite a bit of behavior - is there not a subclass situation that we can use?